docs/integrations/builtin/cluster-nodes/sub-nodes/n8n-nodes-langchain.lmchatgooglevertex.md

```diff
@@ -0,0 +1,36 @@
+---
+#https://www.notion.so/n8n/Frontmatter-432c2b8dff1f43d4b1c8d20075510fe4
+title: Google Vertex Chat Model
+description: Documentation for the Google Vertex Chat Model node in n8n, a workflow automation platform. Includes details of operations and configuration, and links to examples and credentials information.
+contentType: integration
+---
+
+# Google Vertex Chat Model
+
+Use the Google Vertex AI Chat Model node to use Google's Vertex AI chat models with conversational agents.
+
+On this page, you'll find the node parameters for the Google Vertex AI Chat Model node, and links to more resources.
+
+/// note | Credentials
+You can find authentication information for this node [here](/integrations/builtin/credentials/google/service-account/).
```

Comment:
Just checking: does this _only_ support Google Service Account? No OAuth?

Reply:
Yes, only authentication with a service account is available